packages/flutter/test/material/range_slider_test.dart

```diff
@@ -1409,11 +1398,14 @@ void main() {
     expect(
       valueIndicatorBox,
       paints
-        ..rrect(color: sliderTheme.inactiveTrackColor)
-        ..rect(color: sliderTheme.activeTrackColor)
-        ..rrect(color: sliderTheme.inactiveTrackColor),
+        ..rrect(color: const Color(0x3d2196f3))
+        ..rect(color: const Color(0xff2196f3))
+        ..rrect(color: const Color(0x3d2196f3)),
     );
 
+    expect(valueIndicatorBox, paintsExactlyCountTimes(#drawRect, 1));
```

Comment:
Please add a comment here that explains why we expect 1 drawRect and 2 drawRRects.

Reply:
Updated